docs/en_US/example/concurrency.md

```diff
@@ -0,0 +1,51 @@
+# Notify when current changes
```

Comment:
File name is misleading. How about capture_change_data.md 

Is it better to make the title more generic and demonstrate the specific scenario in the content/abstract.